src/searchByAuthorID.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs'); //like import 
+//const book = require('./book');
+
+//1. Read data from json file (contains array type)
+let rawdata = fs.readFileSync('./src/data/books.json');  
+let bookDetail = JSON.parse(rawdata);  //returns js arrays
+
+
+//returns list book ids 
+function searchBookByAuthorID(AuthorID){
+    var booksByAuthor = [];
+    for(var i = 0; i < bookDetail.length; i++){
+        if (AuthorID == bookDetail[i].author.id){
+            booksByAuthor.push(bookDetail[i].title);
```

Comment:
instead of the book title, we want to return the book details. so change line 14 like this:
booksByAuthor.push(bookDetail[i]);